### ISSUE TITLE
Provide generic Helm chart for shell-operator

### ISSUE THREAD

**Issue by nabokihms** [flant/shell-operator#586]
**Is your feature request related to a problem? Please describe.**
There is no established way to install shell-operator.

**Describe the solution you'd like to see**
The proposed solution is a generic Helm chart designed to facilitate the integration of shell-operator into projects. This Helm chart will offer a set of instructions for straightforward implementation and will be adaptable to accommodate specific project requirements.

**Describe alternatives you've considered**

**Additional context**

**Comment by BrLucianoSilva** [comment]
Indeed, it is totally necessary and the current helm chart in the example section is outdated.
